10up/simple-podcasting
# Revamped Podcast Block (In progress yet)

### Description of the Change
Podcast player UI revamped


Closes #191 

### Verification Process

- Add a podcast block to a post or page
- Save that and visit, the enhanced one should appear

### Checklist:






### Changelog Entry



### Credits


Props @jayedul
